### PR TITLE
Features/section508

### PR DESCRIPTION
**Pull request by chris-c-thomas**
**Section 508**
* Global search ARIA implementation
* Contrast Updates among links/headers

### CODE REVIEW

File: public/js/p3/widget/templates/GlobalSearch.html [hunk 1]
```diff
@@ -16,14 +16,15 @@
...
+					<label id="globalSearch" class="sr-only">Global Search</label>
+					<input aria-labelledby="globalSearch" placeholder="Global Search" style="width:100%;" data-dojo-type="dijit/form/TextBox" data-dojo-attach-event="onChange:onInputChange,keypress:onKeypress" data-dojo-attach-point="searchInput"/>
 				</td>
 				<td style="width:1em;padding:2px;font-size:1em;"><i class="fa fa-1x icon-search" data-dojo-attach-event="click:onClickAdvanced" title="Advanced Search"/></td>
 				<td style="width:1em;padding:2px;font-size:1em;"><i class='fa fa-1x icon-question-circle-o NonmodalDialogButton' rel='help:/misc/GlobalSearchOptions' title="Select search options from the dropdown list. Click to view the help information"/></td>
 				<td style="width:50px">
 					<span data-dojo-attach-point="searchOption" data-dojo-type="dijit/form/Select" style="display:inline-block;width:100%">
 						<option selected="true" value="option_and">All terms</option>
-						<option value="option_or">Any term</option>
+						<option value="option_or"</option>
```
Comment (hyoo) [inline public/js/p3/widget/templates/GlobalSearch.html]: this doesn't seem right.

File: views/home.ejs [hunk 2]
```diff
@@ -131,7 +131,7 @@
                     <div style="display:table-cell; width:231px;" class="HomeServiceLink<% if (index < 1) { %> selected <% } %>"
                         rel="<%- svc.rel %>">
                         <H3 class="highlight-b"><%- svc.name %></H3>
-                        <H4><%- svc.secondary %></H4>
+                        <H4 class=""><%- svc.secondary %></H4>
```
Comment (hyoo) [inline views/home.ejs]: what is this for?

File: public/js/p3/resources/Header.css [hunk 3]
```diff
@@ -127,12 +127,16 @@
...
+    color: #ffbc3e;
     font-weight: 600;
     margin-left: 3px;
     /*margin-bottom: -4px;*/
 }
 
+#widget_globalSearch .dijitPlaceHolder {
+    color: #6f6f6f;
+}
+
```
Comment (hyoo) [inline public/js/p3/resources/Header.css]: does this match?

Comment (chris-c-thomas) [reply public/js/p3/resources/Header.css]: It does. Dojo generates this ID on the search input tag.

Comment (hyoo) [reply public/js/p3/resources/Header.css]: thanks for the clarification.

**Pull request merged**